pkg/multiTenancyPlugins/flavors/flavors.go

```diff
@@ -101,9 +101,9 @@ func (flavorsImpl *DefaultFlavorsImpl) Handle(command utils.CommandEnum, cluster
 			}
 		}
 		log.Debug("Plugin flavors apply flavor: ", _key)
-		
+
 		oldconfig.ContainerConfig.HostConfig.Memory = flavors[_key].Memory
```

Comment:
1 - Using underscore is not custom in Go :-)
2 - What if flavors[_key] you will get panic